packages/core/src/prompts/prompt.ts

```diff
@@ -111,6 +115,27 @@ export default class Prompt {
 
 	public prompt() {
 		return new Promise<string | symbol>((resolve, reject) => {
+			if (this.abortController) {
+				if (this.abortController.aborted) {
+					this.state = 'cancel';
+
+					this.input.unpipe();
+					this.input.removeListener('keypress', this.onKeypress);
+					this.output.write('\n');
+					setRawMode(this.input, false);
+					// this.rl.close(); // The readline interface is not set up yet
```

Comment:
can we change the `close` method to do something like `if (this.rl) { this.rl.close(); }`?

then we can call `this.close();` here instead of duplicating these lines

Reply:
Added this in [1557996](https://github.com/bombshell-dev/clack/pull/216/commits/155799643fe1b5bd8cb404af62be128eb7ab3bc7)